lib/elixir/unicode/unicode.ex

```diff
@@ -403,3 +415,87 @@ defmodule String.Graphemes do
     {acc, string}
   end
 end
+
+defmodule String.Normalizer do
+  @moduledoc false
+
+  decomposition_path = Path.join(__DIR__, "Decomposition.txt")
+
+  decompositions = Enum.reduce File.stream!(decomposition_path), [], fn(line, acc) ->
+    [key, first, second, third, fourth, _] = :binary.split(line, ";", [:global])
+    decomposition =
+      String.Hex.to_binary(first) <> (String.Hex.to_binary(second) || "") <>
+                                     (String.Hex.to_binary(third)  || "") <>
+                                     (String.Hex.to_binary(fourth) || "")
+    [{String.Hex.to_binary(key), decomposition} | acc]
+  end
+
+  combining_class_path = Path.join(__DIR__, "CombiningClasses.txt")
+
+  combining_classes = Enum.reduce File.stream!(combining_class_path), [], fn(line, acc) ->
+    [codepoint, class, _] = :binary.split(line, ";", [:global])
+    [{String.to_integer(codepoint, 16), class} | acc]
+  end
+
+  # Compare
+
+  def equivalent?(string, another_string) do
```

Comment:
We can probably move this function to String. There is no reason for it to be here now. :)